app/views/password/password_form.html.erb

```diff
@@ -4,20 +4,26 @@
 
 <h1 class="govuk-heading-l"><%= t 'password.change_password_heading' %></h1>
 
-<%= form_for @password_form, url: profile_change_password_path do |f| %>
+<%=error_summary_for(@password_form&.errors, :form) %>
+<%= form_for @password_form, url: profile_change_password_path, as: :form do |f| %>
   <fieldset class="govuk-fieldset" aria-describedby="changed-password-hint">
-    <div class="govuk-form-group">
-      <%= f.label :old_password, t('password.current_password_lbl'), class: "govuk-label" %>
-      <%= f.password_field :old_password, class: "govuk-input govuk-input--width-20" %>
+    <div class="govuk-form-group <%= 'govuk-form-group--error' if @password_form.errors.key?(:old_password) %>">
+      <h3 class="govuk-heading-m"><%= f.label :old_password, t('password.old_password_lbl'), class: "govuk-label" %></h3>
+      <%= error_message_on(f.object.errors, :old_password) %>
+      <%= f.password_field :old_password, autofocus: true, class: "govuk-input govuk-input--width-20" %>
```

Comment:
the field should have a conditional class `govuk-input--error` to add the red border around it

Reply:
once fixed, update the screenshot in the PR please